tests/ha/check_cluster.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Cluster status and check logs to find error
+# Maintainer: Loic Devulder <ldevulder@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    barrier_wait("FENCING_DONE_$cluster_name");
+
+    # We need to be sure to be root and, after fencing, the default console on node01 is not root
+    select_console 'root-console';
+
+    # Do some extra verification and export logs
+    assert_script_run '(( $(grep -sR segfault /var/log | wc -l) == 0 ))';
+    assert_script_run 'crm script run health';
+    ha_export_logs;
+
+    barrier_wait("LOGS_CHECKED_$cluster_name");
+}
+
+# Specific test_flags for this test module
+sub test_flags {
+    # 'fatal'          - abort whole test suite if this fails (and set overall state 'failed')
+    # 'ignore_failure' - if this module fails, it will not affect the overall result at all
+    # 'milestone'      - after this test succeeds, update 'lastgood'
+    # 'norollback'     - don't roll back to 'lastgood' snapshot if this fails
+    return {milestone => 1, ignore_failure => 1};
```

Comment:
that status "ignore_failure" has been included by fedora guys on special request when we (SUSE/openSUSE) agreed we want any module failure to fail the job without exceptions. So I strongly object that flag to be used. If you know about an issue that you want to workaround after detecting use `record_soft_failure`

Reply:
I thought we agreed to use non-fatal, which is there by default. So it will show module as failed, but won't prevent next test modules execution.

Reply:
Depends on what you expect. If the test itself is non-fatal - use non-fatal (duh). For known bugs I can only agree on @okurz to track them with a record_soft_failure (if you already know a bug will happen because of $reason).

So @ldevulder; why did you set this flag here? What exactly do you want to catch/avoid? A bug you discovered while developing this test?